apps/src/utils.js

```diff
@@ -1,5 +1,6 @@
-/* global define, $ */
-import Immutable from 'immutable';
+// @flow
```

Comment:
This is essentially how you scope. Flow will still look at the imports and do basic type inference. So really it *is* on for everything in src/ and the `// @flow` comment just tells flow to warn about missing types in this file.